cli/src/workers.ts

```diff
@@ -52,14 +55,24 @@ export function buildWatcher<T extends WorkerScript>({
 
   watcher
     .on('add', async (path) => {
-      modules[path] = await compile(path);
+      console.log(`[watcher] add ${path}`);
```

Comment:
Nitpick, what about having a helper function so we don't repeat `console.log("[watcher] "`?

Reply:
Yeah, I don't like adding those logs there. I am adding a lot of logs everywhere to help debugging with beta users, but this will be removed and refactored in a different way.